app/views/buyers/applications/new.html.slim

```diff
@@ -0,0 +1,8 @@
+- if @account
+  = render '/buyers/accounts/menu'
+
+h2 New Application
+
+div id="new-application-form" data=(new_application_form_metadata(provider: current_account, buyer: @buyer, cinstance: @cinstance))
+  = javascript_pack_tag 'new_application'
+  = javascript_pack_tag 'PF4Styles/application'
```

Comment:
Those JS are dependent on the wrapping div??

Reply:
First one loads the React component, whereas the second adds Patternfly styles.